LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Caleb Marble
```

Comment:
Could change this to include both our names

Reply:
oh shoot, i ddin't see that